src/main/scala/com/atomist/rug/spi/CommonViewOperations.scala

```diff
@@ -37,32 +35,6 @@ trait CommonViewOperations[T] extends MutableView[T] with LazyLogging {
     Console.println(msg)
   }
 
-  @ExportFunction(readOnly = true,
-    description = "Report a minor problem")
-  final def minorProblem(@ExportFunctionParameterDescription(name = "msg",
```

Comment:
Yes. There's superior support in the TypeScript library that automatically includes the line and column when a review comment is made.